### ISSUE TITLE
Reasons for low accuracy

### ISSUE THREAD

**Issue by miziworld** [espnet/espnet#4094]
Hello, Thank you for your always quick and kind reply.
I'm using espnet/egs/ksponspeech/asr1 recipe.
As a result of training 100 epochs with KsponSpeech's data on PC, it looked like there was hardly any training.
The decoded result shows that only 2-3 words are repeated.
There is no change other than changing the number of gpu, number of cores, and batch size, so why is the training not working? Could the performance be degraded so much by the batch size or the number of cores used?

My environment is
----GPU : A100 32G x1
----number of core : 48
----batch_size=32
----epoch: 100
and memory capacity is sufficient
Also, I'm training it on other environment PC, the accuracy seems to stay around 0.5. 
I'm using the default lr=10.0, is it because the value is too small?

I attach my training results.
![image](https://user-images.githubusercontent.com/17849068/154924319-9d8ead8b-206d-41c8-bcc7-7d1f35664c6c.png)
![loss](https://user-images.githubusercontent.com/17849068/154924342-8268e006-62a3-40b2-9fb4-1b9ca2457963.png)
![acc](https://user-images.githubusercontent.com/17849068/154924360-76d5add3-5970-4887-894f-6401bef4da03.png)

**Comment by sw005320** [comment]
Actually, the batch size is an important hyper-parameter to reproduce the same training result.
I recommend you to play with the number of GPU, batch size, and accum_grad parameters to make it in a similar range to that in the recipe.
Also, the curve clearly shows that training was corrupted before the warm up ends. 
So, I recommend you decrease the learning rate (but increasing the "practical" batch size by modifying the number of GPU, batch size, and accum_grad would have a similar effect.
